app/components/code-snippet.css

```diff
@@ -0,0 +1,3 @@
+.code-snippet {
+  margin-top: 0.5rem;
+}
```

Comment:
**suggestion (optional)**

In general, I recommend that reusable components don't set the margin and padding to their container, but let their consumer create an extra `<div>`-element to handle spacing.

Here, the `<CodeSnippet>` component is used in only 1 place, so the benefits of not setting the margin and padding are somewhat limited. But, in case you do want to try out my suggested approach, you can edit the `<GuideSection::Subsection>` component (lines [51-53](https://github.com/ember-learn/ember-octane-vs-classic-cheat-sheet/blob/main/app/components/guide-section/subsection.hbs#L51-L53) and [72-74](https://github.com/ember-learn/ember-octane-vs-classic-cheat-sheet/blob/main/app/components/guide-section/subsection.hbs#L72-L74)) like this:

```hbs
{{! Before }}
{{#each ... as |file|}}
  <CodeSnippet
    @fileName="{{@sectionId}}/{{subsectionId}}/{{file}}"
  />
{{/each}}

{{! After }}
{{#each ... as |file|}}
  <div local-class="code-snippet">
    <CodeSnippet
      @fileName="{{@sectionId}}/{{subsectionId}}/{{file}}"
    />
  </div>
{{/each}}
```

```css
.code-snippet {
  margin-top: 0.5rem;
}

.code-snippet:last-of-child {
  margin-top: 0;  /* I didn't check if this works */
}
```

Reply:
Agree! Adapted.